src/js/player.js

```diff
@@ -355,10 +349,9 @@ function rightClickMiniPlayer(contextImg, contextUrl){
     $.get('templates/miniPlayer.html', (template) => {
         mustache.parse(template);
         const rendered = mustache.render(template, {
-            videoUrl: videoSUrl,
-            videoThumbnail: contextImg
+            videoUrl: video.formatStreams[0].url,
```

Comment:
This URL won't always be valid, especially when changing to `youtubedlGetInfo()`.  Some older videos only max out at 480p and checks are needed to make sure that the video is valid.  There are [several](https://github.com/FreeTubeApp/FreeTube/blob/48436e50b3be179562b9a5ce54006d5152c2df88/src/js/player.js#L346) [different](https://github.com/FreeTubeApp/FreeTube/blob/48436e50b3be179562b9a5ce54006d5152c2df88/src/js/player.js#L92) [checks](https://github.com/FreeTubeApp/FreeTube/blob/48436e50b3be179562b9a5ce54006d5152c2df88/src/js/videos.js#L407) that are being made to ensure that the video plays properly and changes the URL if one doesn't work.

Reply:
So, how do achieve this?